src/components/Contexts/AuthContext.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {
+  FC,
+  createContext,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import {
+  isLoggedIn,
+  logIn
+} from '../../api/authn';
+
+export type ContextState = {
+  isLoggedIn: boolean;
+  user: object;
```

Comment:
Is it possible we could create a type for the user object? This will make it easier to predict the data structure provided by the AuthContext for components using it.

e.g.
```
export type User = {
  firstName: string;
  role: "User" | "FederalLead"; // any other roles supported
  // ... other fields provided by the API
};
```

Reply:
Sounds good, Alec. I know I'll miss more stuff like this as I get used to TypeScript, so please feel welcome to point things out as you see them.